sage_modabvar/abvar.py

```diff
@@ -646,15 +646,18 @@ def elliptic_curve(self):
             raise RuntimeError("Elliptic curve not found" +
                                " in installed database")
 
-        sturm = ModularSymbols(N, 2).sturm_bound()
         a = lambda p: f.modular_symbols(1).eigenvalue(p, 'a')
```

Comment:
> Just a remark: Computing ModularSymbols(N, 2).sturm_bound() isn't hard -- it's a trivial arithmetic function of the weight and level. What is hard is "computing [Hecke eigenvalues] up to the Sturm bound for f...".

Okay. The a(p) code gets called once for each elliptic curve so I checked that the result of a(p) was cached.